test/unit/spec/common/jobManager-Spec.js

```diff
@@ -2007,17 +2007,30 @@ define([
             expect(ctx.jobManagerInstance.looper).toBeDefined();
             expect(ctx.jobManagerInstance.looper.pollInterval).toEqual(TEST_POLL_INTERVAL);
 
-            // should have emitted a request for job status
-            expect(ctx.jobManagerInstance.bus.emit.calls.allArgs()).toEqual([
-                [jcm.MESSAGE_TYPE.STATUS, { [jcm.PARAM.JOB_ID]: jobId }],
-            ]);
             // should have the job data in exec.jobState and exec.jobs.byId
             const jobState = ctx.jobManagerInstance.model.getItem('exec.jobState');
             expect(jobState).toBeDefined();
             expect(jobState.job_id).toEqual(jobId);
             expect(jobState).toEqual(
                 ctx.jobManagerInstance.model.getItem(`exec.jobs.byId.${jobId}`)
             );
+
+            if (jobState.meta && jobState.meta.terminal) {
+                // should not have emitted any status requests
+                expect(ctx.jobManagerInstance.bus.emit.calls.allArgs()).toEqual([]);
+                expect(
+                    ctx.jobManagerInstance.handlers[jcm.MESSAGE_TYPE.STATUS].jobStatusLooper
+                ).not.toBeDefined();
```

Comment:
ensure that terminal jobs don't emit a status request on startup and that the job looper isn't installed